repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py

```diff
@@ -247,10 +245,10 @@ def platform_check():
             title='How to in-place upgrade SAP environments from RHEL 7 to RHEL 8'),
         '9': reporting.ExternalLink(
             url='https://red.ht/how-to-in-place-upgrade-sap-environments-from-rhel-8-to-rhel-9',
-            title='How to in-place upgrade SAP environments from RHEL 8 to RHEL 9'),
+            title='Upgrading SAP environments from RHEL 8 to RHEL 9'),
         '10': reporting.ExternalLink(
             url='https://red.ht/how-to-in-place-upgrade-sap-environments-from-rhel-9-to-rhel-10',
```

Comment:
the title change presumes the link change too. Thefore we have 404 on it now. 
plus it seems that shortcut was applicable to the previous doc portal, but not new one. 

The correct link for 8to9 is https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_sap_solutions/9/html/upgrading_sap_environments_from_rhel_8_to_rhel_9/index

and similarly for 7to8 https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_sap_solutions/8/html/upgrading_sap_environments_from_rhel_7_to_rhel_8/index

Reply:
and the same would be for 9to10, ie. https://docs.redhat.com/en/documentation/red_hat_enterprise_linux_for_sap_solutions/10/html/upgrading_sap_environments_from_rhel_9_to_rhel_10/index

Reply:
The links are changed outside of the code, we just change the redirects. This is so that we can change the redirects anytime and don't have to wait for a release.

Reply:
@alexxa yeah, about changed target URLs we know already. We will be fixing them during the next week together with other shortened URLs. But it seems that title should be updated also for 7 -> 8

Reply:
Yes I will update 7->8

Reply:
and shortened URLs are expected to stay same. we will update only targets where they should point to.